ReactNativeClient/lib/models/Setting.js

```diff
@@ -334,7 +335,36 @@ class Setting extends BaseModel {
 			'encryption.passwordCache': { value: {}, type: Setting.TYPE_OBJECT, public: false, secure: true },
 			'style.zoom': { value: 100, type: Setting.TYPE_INT, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Global zoom percentage'), minimum: 50, maximum: 500, step: 10 },
 			'style.editor.fontSize': { value: 13, type: Setting.TYPE_INT, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Editor font size'), minimum: 4, maximum: 50, step: 1 },
-			'style.editor.fontFamily': { value: '', type: Setting.TYPE_STRING, public: true, appTypes: ['desktop'], section: 'appearance', label: () => _('Editor font family'), description: () => _('This must be *monospace* font or it will not work properly. If the font is incorrect or empty, it will default to a generic monospace font.') },
+			'style.editor.fontFamily': {
+				value: '',
+				type: Setting.TYPE_STRING,
+				public: true,
+				appTypes: ['desktop'],
+				section: 'appearance',
+				label: () => _('Editor font family'),
+				description: () =>
+					_('This must be *monospace* font or it will not work properly. If the font ' +
+					'is incorrect or empty, it will default to a generic monospace font.'),
+			},
+			editorFont: {
```

Comment:
For consistency, is it possible to reuse the style.editor.fontFamily key? You could conditionally set some properties like isEnum depending on whether we're on mobile or desktop. Then I guess the property would need to be a string, which I think is acceptable? The value would just be the actual font name "Avenir", "Menlo", etc. instead of the constants.

Reply:
Oh good catch! I completely missed `style.editor.fontFamily` above. 🤦‍♀ Thank you, I've just pushed a commit fixing that.

Reply:
Looks like the commit isn't here yet? Also note that there's an issue with eslint where it will unstage files when making changes, but not staging them back.

Reply:
Whoops here you go: https://github.com/laurent22/joplin/pull/1797/commits/5c474caa678d29a00e7896cda101079638680efc